### PR TITLE
HESS source data

### PR DESCRIPTION
**Pull request by pdeiml**
I added some information and dummy data files for HESS sources.

### CODE REVIEW

File: input/sources/tev-000065.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
+
 source_id: 65
 
 common_name: HESS J1418-609
 gamma_names: [HESS J1418-609]
 fermi_names: []
 other_names:
```
Comment (pdeiml) [inline input/sources/tev-000065.yaml]: In the HESS source catalog HESS J1418-609 corresponds to Rabbit and G313.3+0.1 and not to Kookaburra.  -> See comment in source 66 as well

File: input/sources/tev-000066.yaml [hunk 2]
```diff
@@ -4,7 +4,8 @@ common_name: HESS J1420-607
 gamma_names: [HESS J1420-607]
 fermi_names: []
 other_names:
-- PSR J1420-6048
```
Comment (pdeiml) [inline input/sources/tev-000066.yaml]: In HESS source catalog HESS J1420-607 corresponds to Kookaburra -> See comment on source 65 as well

**Comment by pdeiml** [comment]
ApSS 309 (2007) 277
This reference is mentioned for source 119 but I don't know whether we want to add it to the list in /input/sources/tev-000119.yaml or not.

File: input/sources/tev-000147.yaml [hunk 3]
```diff
@@ -26,6 +26,8 @@ pos:
 reference_ids:
```
Comment (pdeiml) [inline input/sources/tev-000147.yaml]: On the hess source catalog webpage there is also mentioned AICP 848 (2006) 570 as reference. Do we want to add this reference?

**Comment by pdeiml** [comment]
Review required.

File: input/sources/tev-000065.yaml (hunk 1 above)
Comment (cdeil) [reply input/sources/tev-000065.yaml]: I think "Kookaburra" isn't really a well-defined thing. The best explanation I could find is in
https://www.aanda.org/articles/aa/full/2006/34/aa5511-06/aa5511-06.html
which says it's a complex region and both HESS J1418 and J1420 are in the Kookaburra region.
So I'm putting "Kookaburra" in `other_names` for both.

File: input/sources/tev-000066.yaml (hunk 2 above)
Comment (cdeil) [reply input/sources/tev-000066.yaml]: Thanks for spotting this. Yes, HESS J1420 should also have "Kookaburra" in `other_names`

File: input/sources/tev-000094.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ gamma_names: [HESS J1708-443]
 fermi_names: []
 other_names:
 - PSR B1706-44
-- G343.1-2.3
+- G343.1-1.3
```
[the remark this answers is not in the document]
Comment (cdeil) [reply input/sources/tev-000094.yaml]: I think this is a typo in https://www.mpi-hd.mpg.de/hfm/HESS/pages/home/sources/
This is correct: http://www.physics.umanitoba.ca/snr/SNRcat/SNRrecord.php?id=G343.1m02.3
Changing back to "2.3" and sending an email to Konrad to fix the HESS source list typo.

**Pull request merged**